documentation/dev/spec/protocol/inventory.md

```diff
@@ -529,10 +529,11 @@ title: Inventory protocol
           <!ELEMENT REGVALUE (#PCDATA)>
           <!ELEMENT HIVE (#PCDATA)>
 
-        <!ELEMENT RUDDER (AGENT, UUID, HOSTNAME)>
+        <!ELEMENT RUDDER (AGENT, UUID, HOSTNAME SERVER_ROLES)>
```

Comment:
Good shot, just fixed myself.
Sorry in advance, a [s] is missing in the commit message :D